src/domain/disjointDomain.ml

```diff
@@ -117,6 +117,7 @@ struct
       add e acc
     ) (empty ()) es
   let elements m = fold List.cons m [] (* no intermediate per-bucket lists *)
+  let to_seq m = fold Seq.cons m Seq.empty
```

Comment:
Might be worth leaving a comment here that the implementations here aren't efficient: they eagerly fold over the set to construct the entire sequence, as opposed to lazily constructing the sequence.